src/main/groovy/ru/qatools/selenoud/AbstractCloud.groovy

```diff
@@ -150,7 +151,7 @@ abstract class AbstractCloud implements Cloud {
         inputStreamToResponse(logCollector.get(sessionId.split(':')[0]), response)
     }
 
-    protected abstract Container launchContainer(String browserName, String browserVersion, String name)
+    protected abstract Container launchContainer(String browserName, String browserVersion, String name, Map options)
```

Comment:
I don't have a lot of experience in Groovy, but shouldn't this Map be typed like in Java?